elementary/monitor/data_monitoring.py

```diff
@@ -68,18 +67,34 @@ def __init__(
     ):
         self.config = config
         self.tracking = tracking
-        self.dbt_runner = DbtRunner(
+        self.internal_dbt_runner = DbtRunner(
             dbt_project_utils.PATH,
             self.config.profiles_dir,
             self.config.profile_target,
             dbt_env_vars=self.config.dbt_env_vars,
         )
-        self.tests_api = TestsAPI(dbt_runner=self.dbt_runner)
-        self.models_api = ModelsAPI(dbt_runner=self.dbt_runner)
-        self.sidebar_api = SidebarAPI(dbt_runner=self.dbt_runner)
-        self.lineage_api = LineageAPI(dbt_runner=self.dbt_runner)
-        self.filter_api = FiltersAPI(dbt_runner=self.dbt_runner)
+        if self.config.project_dir:
+            self.user_dbt_runner = DbtRunner(
+                self.config.project_dir,
+                self.config.profiles_dir,
+                self.config.profile_target,
+                dbt_env_vars=self.config.dbt_env_vars,
+            )
+        else:
+            self.user_dbt_runner = None
+            logger.warning(
```

Comment:
Haven't we talked about cloud users that we don't want to tell the that they must provide a local dir yet?